src/rpc/rawtransaction.cpp

```diff
@@ -889,7 +888,7 @@ static RPCHelpMan testmempoolaccept()
                 },
                 RPCResult{
                     RPCResult::Type::ARR, "", "The result of the mempool acceptance test for each raw transaction in the input array.\n"
-                        "Length is exactly one for now.",
+                        "Length is exactly one if any failures occur.",
```

Comment:
Do we want to commit to that?

Reply:
I believe we plan to validate packages atomically, so we would never have a situation of "these txns passed, but those didn't." That's why I believed length 1 for failure would be alright, especially since this wouldn't be an API-breaking change. I imagine it could be helpful to return more information to the client, but don't know what that would look like concretely.

Reply:
(Open to ideas)